.github/actions/install-dependencies/action.yml

```diff
@@ -0,0 +1,33 @@
+name: 'Install dependencies'
+description: 'Install dependencies to build sssd'
```

Comment:
Perhaps add this to a bash script under ./scripts so it can be used also outside actions?

Reply:
While doing your proposal I stumbled on a script that is already doing that. Can you check and see if you agree with the newest version?